testing/src/main/java/org/oppia/android/testing/story/StoryProgressTestHelper.kt

```diff
@@ -1083,7 +1083,7 @@ class StoryProgressTestHelper @Inject constructor(
   }
 
   /**
-   * Marks all lessons as recently played. For specifics on parameters and nuances, see:
+   * Marks all lessons as Started_not_completed. For specifics on parameters and nuances, see:
```

Comment:
```suggestion
   * Marks all lessons as started, but not completed. For specifics on parameters and nuances, see:
```

Prefer expanding this out. Ditto for other KDocs.

Reply:
done

Reply:
I don't think this was fully resolved, but I left a newer comment.